Dockerfile

```diff
@@ -23,6 +23,11 @@ RUN chmod +x docker-entrypoint.sh && \
     mkdir /.cache && chmod 777 /.cache
 
 COPY favicon ./favicon
+COPY docker-entrypoint.sh .
+
+# Use sed to strip carriage-return characters from the entrypoint script (in case building on Windows)
+RUN sed -i 's/\r$//g' docker-entrypoint.sh && chmod +x docker-entrypoint.sh

```

Comment:
Same goes for the separate RUN command, better to have all of them in one command.

Reply:
See above. I'll merge it back in.